pages/matvarden.vue

```diff
@@ -1,39 +1,45 @@
 <template>
   <v-container>
-    <div id="map" class="mx-auto text-center">
-      <client-only>
-        <l-map :zoom="13" :center="center" style="height: 80vh; width: 80vw;">
-          <l-tile-layer url="https://{s}.tile.osm.org/{z}/{x}/{y}.png" />
-          <l-marker :lat-lng="center" />
-          <l-circle :lat-lng="center" :radius="radius" />
-        </l-map>
-      </client-only>
-    </div>
+    <div id="newmap" style="height: 80vh;" />
   </v-container>
 </template>
 
 <script>
+import axios from 'axios'
+
 export default {
-  data () {
-    return {
-      center: [62.3908, 17.3069],
-      radius: 0
-    }
-  },
   mounted () {
     const L = this.$L
     const component = this
 
-    const mymap = L.map('map')
+    const newmap = L.map('newmap')
+
+    L.tileLayer('https://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+      attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+      maxZoom: 18
+    }).addTo(newmap)
+
+    newmap.on('locationfound', onLocationFound)
+
+    newmap.locate({ setView: true, watch: true, enableHighAccuracy: true })
 
     function onLocationFound (e) {
       component.center = e.latlng
       component.radius = e.accuracy
     }
 
-    mymap.on('locationfound', onLocationFound)
-
-    mymap.locate({ setView: true, watch: true, enableHighAccuracy: true })
+    axios({
+      method: 'GET',
+      url: 'https://iotsundsvall.northeurope.cloudapp.azure.com/api/graphql?query={snowdepths{from{pos{lat,lon}}when,depth}}'
```

Comment:
Can we get the API URL as an environment setting? Not required for PR, but would be a good feature for later.